sa/authz.go

```diff
@@ -15,8 +16,114 @@ var authorizationTables = []string{
 	"pendingAuthorizations",
 }
 
+var pendingStatuses = []core.AcmeStatus{
+	core.StatusPending,
+	core.StatusProcessing,
+	core.StatusUnknown,
+}
+
 const getAuthorizationIDsMax = 1000
 
+func statusIsPending(status core.AcmeStatus) bool {
+	for _, pendingStatus := range pendingStatuses {
```

Comment:
I was torn initially but with only three statuses I thought the readable `range` approach was better.